### ISSUE TITLE
AttributeError: 'int' object has no attribute 'replace'

### ISSUE THREAD

**Issue by LootScooper** [regosen/gallery_get#63]
Traceback (most recent call last):
  File "/mnt/4tb/GGDL/gallery_get.py", line 334, in run
    self.run_internal()
  File "/mnt/4tb/GGDL/gallery_get.py", line 320, in run_internal
    self.process_redirect_page(info, response)
  File "/mnt/4tb/GGDL/gallery_get.py", line 296, in process_redirect_page
    (info.path,info.subtitle) = safe_unpack(jpegs[0],info.subtitle)
  File "/mnt/4tb/GGDL/gallery_get.py", line 67, in safe_unpack
    return (obj[0],safe_str(obj[1]))
  File "/mnt/4tb/GGDL/gallery_get.py", line 58, in safe_str
    name = name.replace(":",";") # to preserve emoticons
AttributeError: 'int' object has no attribute 'replace'
Using params: ['core.py']

**Comment by regosen** [comment]
I'm not sure which site you used, but we just fixed a number of plugins this week.  Can you try again with the latest version? (1.8.10)

**Comment by LootScooper** [comment]
i will give it a try

**Comment by youngdarwin** [comment]
I'm getting this error with the latest version (downloaded a fresh copy today). See traceback below. I tried to debug this but it's beyond me right now. Parameter 'name' is 'jpg' until an integer value of 104 appears and causes this crash. Because I don't know what safe_str() is doing, it's hard to know what this means. I consistently get this error on big imagefap galleries, but not at the same point, I think, so I think maybe IF starts returning error codes. 

Traceback (most recent call last):
  File "gallery_get.py", line 334, in run
    self.run_internal()
  File "gallery_get.py", line 320, in run_internal
    self.process_redirect_page(info, response)
  File "gallery_get.py", line 296, in process_redirect_page
    (info.path,info.subtitle) = safe_unpack(jpegs[0],info.subtitle)
  File "gallery_get.py", line 67, in safe_unpack
    return (obj[0],safe_str(obj[1]))
  File "gallery_get.py", line 58, in safe_str
    name = name.replace(":",";") # to preserve emoticons
AttributeError: 'int' object has no attribute 'replace'
Using params: ['gallery_get.py', 'https://www.imagefap.com/pictures/7014092/Sitting-nudes-2', 'img']

When I added "print(name)" at the top of safe_str, I got these lines (at the end of hundreds of lines):

...
Skipping existing file: https://cdn.imagefap.com/images/full/64/170/1705868360.jpg?end=1583869796&secure=0afe991d7771df8aa3e4c
jpg
https://cdn.imagefap.com/images/full/64/468/468985703.jpg?end=1583869797&secure=08234d3fe03f13cc4d3c1 -> img/Sitting nudes 2 Porn Pics & Porn GIFs/468985703.jpg
jpg
https://cdn.imagefap.com/images/full/64/165/165439597.jpg?end=1583869798&secure=05d8cf7198bb7b3ab08d4 -> img/Sitting nudes 2 Porn Pics & Porn GIFs/165439597.jpg
104
